src/components/Calendar/Calendar.tsx

```diff
@@ -0,0 +1,198 @@
+import { Grid, makeStyles } from '@material-ui/core';
+import clsx from 'clsx';
+import generateScheduledEventFilter from 'filters/scheduledEvent/scheduledEventsFilter';
+import moment from 'moment';
```

Comment:
@peter-asztalos I see you are using here moment and in the backend luxon. Is that for some reason or we can use luxon on the fontend as well?

Reply:
The calendar library provides localization using different kind of libs, moment is one of them currently, so I decided on moment as I think many of us is already familiar with it's API and provides wide range of function for other uses cases also.